_stbt/control.py

```diff
@@ -557,6 +558,24 @@ def lirc_key_reader(cmd_iter, control_name):
             yield m.group('key')
 
 
+def lircd_remote_listen(lircd_socket=None):
```

Comment:
I found the difference between `lircd_remote_listen` and `lirc_remote_listen` quite confusing, so I've pushed a fixup to the docstrings and the man page.